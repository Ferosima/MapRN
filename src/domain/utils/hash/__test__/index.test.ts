import { hash } from "..";

describe("hash", () => {
  it("should always generate same hash string", () => {
    expect(hash({ a: 1, b: [2] })).toMatchSnapshot();
  });
});
