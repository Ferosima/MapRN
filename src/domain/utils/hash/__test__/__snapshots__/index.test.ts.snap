// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hash should always generate same hash string 1`] = `"-735355204"`;
